satsale.py

```diff
@@ -102,7 +108,8 @@ def pay():
 
 @api.doc(
     params={
-        "amount": "An amount in USD.",
+        "amount": "An amount in `config.base_currency`.",
+        "currency": "Currency code `config.base_currency`",
```

Comment:
I think this is "currency" line is unnecessary here, we haven't yet set it up as an argument for /pay?currency=.

Reply:
True. Until this argument is implemented, this param is redundant. 

I removed it